db/migrate/20230921181155_add_territory_id_to_services.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTerritoryIdToServices < ActiveRecord::Migration[7.0]
+  def change
+    add_column :services, :territory_id, :integer
```

Comment:
Avec `add_reference` on ajoute aussi une foreign key et un index. :wink: 

```suggestion
    add_reference :services, :territory, foreign_key: true, index: true
```